lib/ansible/modules/source_control/github_release.py

```diff
@@ -145,6 +206,21 @@ def main():
         else:
             module.exit_json(tag=None)
 
+    if action == 'create_release':
+        if tag is None:
+            module.fail_json(msg="Specify tag in order to create a release.")
```

Comment:
I've implemented it here: 4399c301a2413f394e78bb048656b9cf744940e9
Please let me know if that's right.